deploy/example_catalog/cr-app-training-engine.json

```diff
@@ -0,0 +1,152 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name": "training-engine"
+    },
+
+    "spec": {
+        "categories": [
+            "AIML/Training"
+        ],
+        "config": {
+            "configMeta": {
+                "ml_engine": "python"
+            },
+            "roleServices": [
+                {
+                    "roleID": "RESTServer",
+                    "serviceIDs": [
+                        "gunicorn",
+                        "ssh"
+                    ]
+                },
+                {
+                    "roleID": "LoadBalancer",
+                    "serviceIDs": [
+                        "gunicorn",
+                        "haproxy-stats",
+                        "haproxy-train"
+                    ]
+                },
+                {
+                    "roleID": "controller",
+                    "serviceIDs": [
+                        "ssh",
+                        "jupyter-nb"
+                    ]
+                }
+            ],
+            "selectedRoles": [
+                "RESTServer",
+                "LoadBalancer",
+                "controller"
+            ]
+        },
+        "configSchemaVersion": 7,
```

Comment:
verify if we still need this **configSchemaVersion**

Reply:
verified that it is required